e2e-tests/commands/insurance/complete-prepare-application-section-single-policy-type.js

```diff
@@ -26,7 +26,7 @@ const completePrepareApplicationSinglePolicyType = ({
   cy.completeBusinessSection({ differentTradingAddress, submitCheckYourAnswers });
 
   cy.completeBuyerSection({
-    viaTaskList: false,
+    viaTaskList: true,
```

Comment:
Can we instead, remove this and update the command to have a default `viaTaskList` value? All the other sections do this